### ISSUE TITLE
composer require civicrm/cv failing on Drupal 8.9.x

### ISSUE THREAD

**Issue by jonathandhn** [civicrm/cv#80]
- _civicrm/cv v0.3.3 requires symfony/filesystem ~2.8|^3_
- _Conclusion: remove symfony/filesystem v4.4.11_

**Comment by VladimirAus** [comment]
For Drupal 9
```
    - Conclusion: don't install symfony/console v4.4.9
    - civicrm/cv v0.3.3 requires symfony/console ~2.8|^3
```

**Comment by MikeyMJCO** [comment]
There's an open PR for this. Please test and feedback there.

#76

**Comment by totten** [comment]
Previously resolved. (IIRC, #76 was rebased+extended then merged via #103).
